test/integration/configuration/test_auth_remote_plugin.py

```diff
@@ -0,0 +1,59 @@
+import os
+import textwrap
+
+import pytest
+
+from conan.test.utils.tools import TestClient
+from conans.util.files import save
+
+
+class TestAuthRemotePlugin:
+    """ Test when the plugin fails, we want a clear message and a helpful trace
+    """
+    def test_error_auth_remote_plugin(self):
+        c = TestClient(default_server_user=True)
+        auth_plugin = textwrap.dedent("""\
+            def auth_remote_plugin(remote, user=None, password=None):
```

Comment:
```suggestion
            def auth_remote_plugin(remote, user=None):
```

And the below tests that still have the password argument